### ISSUE TITLE
Static attribute in RoutingTable

### ISSUE THREAD

**Issue by agentgt** [baswerc/routes#3]
Love the library. I have been looking for a something simple that wraps the existing servlet API so that I can make simple shareable Admin interfaces that work across various JVM apps. Many libraries provide a minimal health check or admin UI (e.g. Codahale metrics) off the existing servlet API but this obviously gets tedious with the servlet API.

The problem is this library uses a static singleton in RoutingTable so if I package up multiple libraries that just want to basically make an easier to use Servlet this does not work well (that is it will conflict with other Admin UIs if I use this library in other places).

**Comment by baswerc** [comment]
Yeah that's definitely a shortcoming. I've got some thoughts on how to remove the singleton bridge that I'm planning on including in the next release.
